lib/packwerk/cli.rb

```diff
@@ -21,7 +21,10 @@ def initialize(run_context: nil, configuration: nil, out: $stdout, err_out: $std
       @err_out = err_out
       @style = style
       @configuration = configuration || Configuration.from_path
-      @run_context = run_context || Packwerk::RunContext.from_configuration(@configuration)
+      @run_context = run_context || Packwerk::RunContext.from_configuration(
+        @configuration,
+        reference_lister: ::Packwerk::CheckingDeprecatedReferences.new(@configuration.root_path),
```

Comment:
Nice! I think we should add a `require "packwerk/checking_deprecated_references"` in this file now.